logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LoggedEventImpl.java

```diff
@@ -127,33 +93,14 @@ public int getValueOffset() {
   public int getValueLength() {
     final short metadataLength = getMetadataLength();
 
-    return getMessageLength() - headerLength(metadataLength);
+    return getMessageLength() - LogEntryDescriptor.headerLength(metadataLength);
   }
 
   @Override
   public void readValue(final BufferReader reader) {
     reader.wrap(buffer, getValueOffset(), getValueLength());
   }
 
-  @Override
-  public String toString() {
```

Comment:
:question: Why remove the `toString` method? It's nice to have a string representation. Was it causing any issues?

Reply:
Re-added, although not all of the fields exist anymore so the toString is showing a bit less.